code/nebula/neb.cpp

```diff
@@ -163,22 +163,32 @@ void neb2_get_eye_orient(matrix *eye_matrix);
 // NEBULA FUNCTIONS
 //
 
-// initialize neb2 stuff at game startup
-void neb2_init()
+static poof_info* get_nebula_pointer(char* nebula_name)
```

Comment:
Minor, but specify that this function is referring to poofs, specifically. `get_nebula_poof_pointer` or something.

Reply:
Done